apps/dashboard/src/components/workflow-editor/url-input.tsx

```diff
@@ -46,11 +48,7 @@ export const URLInput = ({
                       <Editor
                         fontFamily="inherit"
                         placeholder={placeholder}
-                        extensions={[
-                          liquid({
-                            variables,
-                          }),
-                        ]}
+                        extensions={[autocompletion({ override: [completions(variables)] })]}
```

Comment:
Since this is the URL input, we shouldn't autocomplete any Liquid filters as they don't make sense in the context of a url-input. Autocomplete for this control should only include, payload, subscriber and previous step values only if there is a previous step (this should be handled by the server).